scripts/ec2cleanup.py

```diff
@@ -0,0 +1,83 @@
+from datetime import datetime
+from utils.providers import list_providers, get_mgmt
+import argparse
+import sys
+from time import sleep
+from utils.log import logger
+
+
+def parse_cmd_line():
+    parser = argparse.ArgumentParser(argument_default=None)
+    parser.add_argument('--max-hours', dest='maxhours', type=int, default=24, help='Max hours '
+        'since Instanced was created. (Default is 24 hours.)')
+    parser.add_argument('--exclude-instances', nargs='+', help='List of instances, '
+        'which should be excluded.')
+    parser.add_argument('--exclude-volumes', nargs='+', help='List of volumes, which should be '
+        'excluded.')
+    parser.add_argument('--exclude-eips', nargs='+', help='List of EIPs, which should be '
+        'excluded. Allocation_id or public IP are allowed.')
+    args = parser.parse_args()
+    return args
+
+
+def delete_old_instances(ec2provider, date, maxhours, excluded_instances):
+    deletetime = maxhours * 3600
+    try:
+        for instance in ec2provider.list_vm(include_terminated=True):
+            if excluded_instances and instance in excluded_instances:
+                continue
+            else:
+                creation = ec2provider.vm_creation_time(instance)
+                difference = (date - creation).total_seconds()
+                if (difference >= deletetime):
+                    ec2provider.delete_vm(instance_id=instance)
```

Comment:
lets add log.info('deleting the instance: {}'.format(instance)), so that we know whats deleted in script/job logs.